docs/examples.py

```diff
@@ -1,13 +1,14 @@
 """
 Examples for the tutorial.
 """
-import os
 import io
+import os
 import sys
 
 sys.path.insert(0, os.path.abspath("../python"))
 
 import numpy as np  # noqa: E402
+
 import msprime  # noqa: E402
```

Comment:
Something about this being in the docs has made msprime part of the local set. I think it is the only "error" left. Not looking into it for now.

Reply:
Fixed, wasn't being rearranged as was below code.